### PR TITLE
Update requirements.txt

### PR DESCRIPTION
**Pull request by gaurab-10**
### Short description

It causes the break build by pycairo removing ">" in the reportlab solved the issue

### Resolved issues

Fixes: #688

### CODE REVIEW

**Review by m7salam — APPROVED** [review]

**Comment by m7salam** [comment]
This is causing huge problem with the pycairo dependency issue so please would be great if we can merge this soon

**Comment by luisza** [comment]
I'm sorry, I cannot merge this PR because it doesn't meet the requirements and it has no impact. Additionally, the requirement forces a version of reportlab that is not desired and needs to be updated in other parts, such as:
- setup.py (more important)
- tox.ini (for testing)
- requirements.txt

It is convenient for it to be a line of the type "reportlab>=3.5.53,<4.0" to avoid future problems.

File: requirements.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ html5lib>=1.1
 Pillow>=8.1.1
 pypdf>=3.1.0
 python-bidi>=0.4.2
-reportlab>=3.5.53
+reportlab=3.5.53
```
Comment (luisza) [inline requirements.txt:7]: reportlab>=3.5.53,<4.0

Comment (luisza) [reply requirements.txt:7]: check setup.py and tox.ini

Comment (gaurab-10) [reply requirements.txt]: Made Changes on 
. setup.py (more important)
. tox.ini (for testing)
. requirements.txt

**Comment by luisza** [comment]
This was fixed on v0.2.11 with PR #690 .
Thanks for your help.

**Pull request closed without merging**